Packs/FiltersAndTransformers/Scripts/RemoveMatches/README.md

```diff
@@ -0,0 +1,28 @@
+Removes items from the given value matching the provided list of patterns. If the matchexact Argument is 'yes', direct string compare is used, otherwise the comparison is done using regex.
```

Comment:
```suggestion
Removes the entire items from the given list of values if they match the provided list of patterns. If the match_exact argument is 'yes', direct string compare is used, otherwise the comparison is done using regex.
```